src/org/labkey/test/params/FieldDefinition.java

```diff
@@ -289,12 +289,14 @@ public String toString()
         MultiLine("Multi-Line Text", "Multi-Line Text", "string"),
         Integer("Integer", "Integer", "int"),
         String("Text", "Text (String)", "string"),
-        Subject("Subject/Participant", "Subject/Participant (String)", "string"),
+        Subject("Subject/Participant", "Subject/Participant (String)",
+                "string", "http://www.labkey.org/exp/xml#string", null),
         DateTime("DateTime", "DateTime", "date"), // TODO remove this after GWT designer removed
         DateAndTime("Date Time", "Date Time", "date"),
         Boolean("Boolean", "Boolean", "boolean"),
         Double("Number (Double)", "Number (Double)", "float"), // TODO remove this after GWT designer removed
-        Decimal("Decimal", "Decimal", "float"),
+        Decimal("Decimal", "Decimal", "float",
+            "http://www.labkey.org/exp/xml#float", null),
```

Comment:
Decimal fields don't have a `conceptURI`
```suggestion
        Decimal("Decimal", "Decimal", "float"),
```